gui/default/syncthing/core/syncthingController.js

```diff
@@ -1097,6 +1097,134 @@ angular.module('syncthing.core')
+                case 'disconnected-inactive':
+                    return 'fa-power-off';
+                case 'insync':
+                    return 'fa-check';
+                case 'paused':
+                    return 'fa-pause';
+                case 'syncing':
+                    return 'fa-sync';
+                case 'unused-disconnected':
+                case 'unused-insync':
+                case 'unused-paused':
+                    return 'fa-unlink';
+
+            }
+        };
+
+        $scope.deviceStatusText = function(device) {
+            var text = '';
+            switch ($scope.deviceStatus(device)) {
+                case 'disconnected':
+                    return $translate.instant('Disconnected');
+                case 'disconnected-inactive':
+                    return $translate.instant('Disconnected (Inactive)');
+                case 'insync':
+                    return $translate.instant('Up to Date');
+                case 'paused':
+                    return $translate.instant('Paused');
+                case 'syncing':
+                    text = $translate.instant('Syncing') +
+                    ' (' +
+                    percentFilter($scope.completion[device.deviceID]._total) +
+                    ', ' +
+                    binaryFilter($scope.completion[device.deviceID]._needBytes) +
+                    'B)';
+                    return text;
+                case 'unused-disconnected':
+                    return $translate.instant('Disconnected (Unused)');
+                case 'unused-insync':
+                    return $translate.instant('Connected (Unused)');
+                case 'unused-paused':
+                    return $translate.instant('Paused (Unused)');
+            }
+        };
+
+        $scope.folderStatusIcon = function(cfg) {
+            switch ($scope.folderStatus(cfg)) {
+                case 'clean-waiting':
+                case 'scan-waiting':
+                case 'sync-preparing':
+                case 'sync-waiting':
+                    return 'fa-hourglass-half';
+                case 'cleaning':
+                    return 'fa-recycle';
+                case 'faileditems':
+                case 'localunencrypted':
+                case 'outofsync':
+                    return 'fa-exclamation-circle';
+                case 'idle':
+                case 'localadditions':
+                    return 'fa-check';
+                case 'paused':
+                    return 'fa-pause';
+                case 'scanning':
+                    return 'fa-search';
+                case 'stopped':
+                    return 'fa-stop';
+                case 'syncing':
+                    return 'fa-sync';
+                case 'unknown':
+                    return 'fa-question-circle';
+                case 'unshared':
+                    return 'fa-unlink';
+            }
+        };
+
+        $scope.folderStatusText = function(folder) {
```

Comment:
Forgot to sort this one?  I find the alphabetical order very useful for review ;-)

Reply:
So this is actually kind of alphabetically sorted, however I also followed the order from `$scope.folderStatusIcon`, and there some cases are merged together, which breaks the alphabetical sorting. Here, I just used the exact same order as there.

Do you think that it would be better to sort the two alphabetically separately and not try to follow the exact same order?

Reply:
Yes. I noticed how it follows the same pattern, but purely alphabetical would be my preferred choice when there aren't any fall-throughs.

Reply:
Done.